htdocs/societe/class/societe.class.php

```diff
@@ -784,6 +784,15 @@ class Societe extends CommonObject
 	 */
 	public $multicurrency_code;
 
+	/**
+	 * @var string login
+	 */
+	public $login;
```

Comment:
There is no reason to have a field ->login on a company. This is not into database.
Same for property ->admin